config/default.json

```diff
@@ -29,10 +33,19 @@
     "port": 3000,
     "getJobModuleApiURL": "http://localhost:3000/job-module/"
   },
+  "waitingThreshold": 0.40,
+  "increaseWorker": 10,
+  "maxWorker": 10,
```

Comment:
"waitingThreshold": 0.40,
"increaseWorker": 10,
"maxWorker": 10,
 add to symmetricWorker option

Reply:
requested changes are addressed in new commit ***'requested changes in review'*** .
the required fields now moved to **symmetricWorker** option